rust/typeql_lang.rs

```diff
@@ -29,7 +29,9 @@ pub mod query;
 #[macro_use]
 mod util;
 
-pub use builder::{contains, eq, gt, gte, like, lt, lte, neq, not, rel, rule, type_, var};
+pub use builder::{
+    abs, ceil, constant, contains, cvar, eq, floor, gt, gte, like, lt, lte, neq, not, rel, round, rule, type_, vvar,
```

Comment:
Do we also want `min/max`? Double check we have everything we want here.

How many places do we have to update when we add a new function? We should add a piece of documentation to the PR description to make a note of where to change things if we add new functions/operators

Reply:
`min` and `max` are macros and are imported in another way.
Aren't all places needs changing obvious by searching for all mentions of existing functions?